src/services/beerService.tsx

```diff
@@ -1,52 +1,54 @@
 import firebase from "../utils/firebaseConfig";
 
+export type Beer = {
+  id: string,
+  name: string, 
+  style: string,
+  brewery: string
+};
+
 const beerService = {
-  findAll: () => {
+  findAll: (): Array<Beer> => {
     const beersDB = firebase.database().ref("beersDB");
-    let list = [];
+    let list: (object)[] = [];
```

Comment:
we should not use `object` type as it is roughly equivalent to `any`

Reply:
Is it fine ?

```suggestion
   let list: Beer[] = [];
```